util/topgen/templates/plic_all_irqs_test.c.tpl

```diff
@@ -146,8 +146,20 @@ static void peripheral_irqs_enable(void) {
 <%
   if p.name == "aon_timer": continue
 %>\
-  CHECK_DIF_OK(
-      dif_${p.name}_irq_restore_all(${args(p)}, &${p.name}_irqs));
+<%
+  indent = ""
+%>\
+  % if p.inst_name == "uart0":
+<%
+  indent = "  "
+%>\
+  if (kDeviceType == kDeviceSimDV) {
```

Comment:
Can you add a brief comment to the templates saying something like 'skip UART0 tests on Verilator and FPGA platforms since they interfere with test output'?

Reply:
Done, thanks.